libs/packages/components/src/lib/click-outside/click-outside.directive.ts

```diff
@@ -15,11 +15,15 @@ export class SDSClickOutsideDirective {
 
   constructor(private _elementRef: ElementRef) {}
 
-  @HostListener('document:click', ['$event.target'])
-  public onClick(targetElement) {
-    const clickedInside = this._elementRef.nativeElement.contains(targetElement);
+  @HostListener('document:click', ['$event'])
```

Comment:
So just to confirm, the intention is to change how **ALL** autocompletes work so that they do not close when an option is clicked?

Reply:
yes, all multiselect demos should remain open after an item is selected, until there is a click outside.